dts/x86/intel_quark_d2000.dtsi

```diff
@@ -0,0 +1,57 @@
+#include "skeleton.dtsi"
+#include "mem.h"
+
+/ {
+	cpus {
+		cpu@0 {
+			compatible = "intel,quark";
```

Comment:
that's fine, just that the 'cpu' section here is meant for the type of core(s) in the SoC.  So from what you say, they should have the same compatible between quark_d2000 & quark_se_c1000.